code/_onclick/other_mobs.dm

```diff
@@ -66,6 +66,39 @@
 				shadow.visible_message(SPAN_WARNING("[shadow] gives up on trying to climb onto \the [A]!"))
 			return
 
+	//PERK_ABSOLUTE_GRAB
+	if(ishuman(A))
+		if(stats.getPerk(PERK_ABSOLUTE_GRAB) && a_intent == I_GRAB)
+			if(stat || paralysis || stunned || weakened || lying || restrained() || buckled)
```

Comment:
This logic better to be unloaded to a different proc, like LaserEyes mutation bellow did.